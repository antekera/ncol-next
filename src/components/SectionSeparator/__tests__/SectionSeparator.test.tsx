import { render } from '@testing-library/react'

import { SectionSeparator } from '..'

describe('SectionSeparator', () => {
  test('should match snapshots', () => {
    const { container } = render(<SectionSeparator />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
