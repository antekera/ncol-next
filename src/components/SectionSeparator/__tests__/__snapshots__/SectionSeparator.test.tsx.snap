// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionSeparator should match snapshots 1`] = `
<hr
  class="border-slate-200-2 mt-28 mb-24 border-solid"
/>
`;
